packages/rocketchat-livechat/.app/client/views/message.js

```diff
@@ -66,7 +66,15 @@ Template.message.helpers({
 			return agent.name || agent.username;
 		}
 		return this.u.username;
-	}
+	},
+
+	hasAttachments() {
+		return ((this.attachments) && (this.attachments.length > 0));
```

Comment:
none of this parenthesis are required.. please remove them to help readability.

Reply:
Fixed.